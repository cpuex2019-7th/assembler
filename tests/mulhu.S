    li t0, -1
    li t1, -1
    mulhu a0, t0, t1
