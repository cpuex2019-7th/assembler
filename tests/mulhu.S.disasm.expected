lui t0, 0
addi t0, t0, -1
lui t1, 0
addi t1, t1, -1
mulhu a0, t0, t1
